packages/cli/src/commands/generate/outputs/generate_outputs_command.test.ts

```diff
@@ -249,3 +254,101 @@ void describe('generate outputs command', () => {
     assert.match(output, /Arguments .* mutually exclusive/);
   });
 });
+
+void describe('GenerateOutputsCommand error handling', () => {
+  let clientConfigGenerator: ClientConfigGeneratorAdapter;
+  let backendIdentifierResolver: AppBackendIdentifierResolver;
+  let generateOutputsCommand: GenerateOutputsCommand;
+
+  beforeEach(() => {
+    // Mock the dependencies
+    clientConfigGenerator = {
+      generateClientConfigToFile: mock.fn(),
+    } as unknown as ClientConfigGeneratorAdapter;
+
```

Comment:
Why not use the existing `describe` that is already doing all the mock setup

Reply:
Updated it, keeping it consistent
I was planning to create Test suites separate for handling all negative test cases